### ISSUE TITLE
[Errno 56] Socket is already connected

### ISSUE THREAD

**Issue by plavcik** [romis2012/aiohttp-socks#2]
I'm using `Python 3.6.4` and

```
aiohttp==3.3.2
aiohttp-socks==0.1.4
```

When using this function
```
async def get_body(url):                                                         
    conn = SocksConnector(socks_ver=SocksVer.SOCKS4, host='127.0.0.1', port=9050)
    async with aiohttp.ClientSession(connector=conn) as session:                 
        async with session.get(url, timeout=30) as response:                     
            assert response.status == 200, "Status %d is not 200" % response.status
            html = await response.read()                                         
            return url, response.url, response.status, html                      
```
For url without redirect like https://github.com/, it's working well. If there is redirect like http://ibm.cz, I'm getting [Errno 56] Socket is already connected

I also noticed for http://ibm.com [Errno 9] Bad file descriptor

**Comment by romis2012** [comment]
Fixed, please upgrade to aiohttp-socks==0.1.5

**Comment by plavcik** [comment]
Thank you for quick fix. It's working with `aiohttp-socks==0.1.5`
